Line.h

```diff
@@ -11,3 +11,4 @@ class Line :
 	void draw(Gdiplus::Graphics &) override;
 };
 
+// qiqiqiqiqiqiqiqiqiqiqiqifeng!!!!!!!
```

Comment:
There's a red circle here. Meaning that you need a new line at the end of file. Can you change that?